controls/datagrid/keyboard-navigation/maccatalystt.md

```diff
@@ -0,0 +1,50 @@
+---
+title: MacCatalyst
+page_title: .NET MAUI DataGrid Documentation - Keyboard Support on MacCatalyst
+description: Learn more about the available keyboard combinations as part of the supported Telerik UI for .NET MAUI DataGrid accessibility standards.
+position: 1
+slug: datagrid-keyboard-support-mac
+---
+
+# .NET MAUI DataGrid Keyboard Support on WinUI
+
+The Telerik UI for .NET MAUI DataGrid provides keyboard navigation support on `MacCatalyst`.
+
+While applying part of the keyboard keys, you can also change the current cell behavior of the DataGrid component. For more details, see the article on [setting the .NET MAUI DataGrid cells]({%slug datagrid-current-cell%}).
+
+The following table lists the actions and keyboard combinations that are available in the DataGrid:
+
+| Hotkey Combinations  | Action |
+|----------------------|-------------------|
+| `Left Arrow` | Focuses the next cell on the left. The selection is changed as well when in single mode. |
```

Comment:
I suggest using the following version in all four descriptions of this type:

When using single selection mode, the selection also changes.